src/components/auth/Signup.js

```diff
@@ -34,6 +34,12 @@ function Signup({ register, loading, errResponse, token, history }) {
     });
   };
 
+  useEffect(() => {
+    if (localStorage.getItem('codeclan_token')) {
+      history.push('/dashboard');
+    }
+  }, []);
```

Comment:
Codacy found an issue: [React Hook useEffect has a missing dependency: 'history'. Either include it or remove the dependency array.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=6042181)